packages/design-system/src/components/HelpDrawer/HelpDrawer.jsx

```diff
@@ -24,6 +26,45 @@ export class HelpDrawer extends React.PureComponent {
 
   componentDidMount() {
     if (this.headingRef) this.headingRef.focus();
+
+    const eventAction = 'onComponentDidMount';
```

Comment:
This could be condensed to:
`this.sendAnalyticsEvent('onComponentDidMount')`
This is just my personal preference, so I don't feel strongly on it at all